ui/lib/core/addon/components/form-field.hbs

```diff
@@ -247,7 +247,7 @@
     <MaskedInput
       @name={{@attr.name}}
       @value={{or (get @model this.valuePath) @attr.options.defaultValue}}
-      @allowCopy="true"
+      @allowCopy={{if @attr.options.noCopy false true}}
```

Comment:
this should default to true. not ideal (rather it default to false) but working with an old pattern and trying to keep backporting/scope simple.